README.md

```diff
@@ -228,13 +228,31 @@ All Databricks resources bypass policy enforcement at creation time. Databricks
 
 ### Resources that are exempt from policy evaluation
 
+Resource types in the following namespaces are excluded from policy evaluation:
 - Microsoft.Resources/*, except resource groups and subscriptions. 
    - For example, `Microsoft.Resources/deployments` and `Microsoft.Resources/templateSpecs` are not evaluated by policy.
 - Microsoft.Billing/*
 - Microsoft.Capacity/reservationOrders/*
 - Microsoft.Help/*
 - Microsoft.Diagnostics/*
 
+The following resource types are also excluded from policy evaluation:
+- Microsoft.Automation/automationAccounts/jobs
```

Comment:
these are excluded only from brownfield evaluation, not greenfield - so maybe should be moved to the section below - though I'm not sure if they're all due to scalability issues, or due to not following the RPC (not supporting list calls at subscription level for example)

Reply:
@varunkumta are you talking about `Microsoft.Automation/automationAccounts/jobs` only, or all the resource types in this new list?

Reply:
all